core/src/main/java/com/linecorp/armeria/client/ClientFactoryOptions.java

```diff
@@ -184,6 +185,10 @@
     public static final ClientFactoryOption<ProxyConfigSelector> PROXY_CONFIG_SELECTOR =
             ClientFactoryOption.define("PROXY_CONFIG_SELECTOR", ProxyConfigSelector.of(ProxyConfig.direct()));
 
+    public static final ClientFactoryOption<MeterIdPrefix> DEFAULT_METER_ID_PREFIX =
+            ClientFactoryOption.define("DEFAULT_DNS_METER_ID_PREFIX",
+                    new MeterIdPrefix("armeria.client.dns.queries"));
```

Comment:
I think this option is too early to introduce now. Could we hard-code for now and then add later when we get some user demands?

Reply:
Sounds good. I will make the change.